darts/backtesting/backtesting.py

```diff
@@ -360,6 +361,10 @@ def backtest_gridsearch(model_class: type,
         if val_series is None:  # expanding window mode
             backtest_forecast = backtest_forecasting(train_series, model, backtest_start_time, fcast_horizon_n)
             error = metric(backtest_forecast, train_series)
+        elif val_series == 'train':
+            model.fit(train_series)
+            # Use ndarray because casting to TimeSeries takes too much time
+            error = metric(model.fitted_values, train_series.univariate_values())
```

Comment:
As far as I can tell, FourTheta is the only model that possesses the attribute `fitted_values`. So it would be good to have a check here to test whether the given model supports this functionality, right?

Reply:
At least all ExponentialSmoothing have a fitted attribute that can be retrieved (So the current theta model could have one too, I think). If you approve this functionality, it might be interesting to add this attribute in other models.
But yes, a check is necessary.